src/main/java/io/vertx/starter/config/ProjectConstants.java

```diff
@@ -29,4 +29,7 @@
   String ARCHIVE_FORMAT = "archiveFormat";
   String PACKAGE_NAME = "packageName";
   String JDK_VERSION = "jdkVersion";
+  String FLAVOR = "flavor";
+  String MUTINY_FLAVOR = "mutiny";
```

Comment:
I don't believe this is necessary. It can be stored as an enum value.

Reply:
addressed in e098a10